lib/ember-app.js

```diff
@@ -86,6 +86,13 @@ EmberApp.prototype.visit = function(path, options) {
     .then(function(instance) {
       return instance.visit(path, bootOptions);
     })
+    .then(function(instance) {
+      var fastbootInfo = instance.lookup('info:-fastboot');
```

Comment:
We don't have the FastBootInfo object in this context, it was created and registered within the instance in `registerFastBootInfo`, then "forgotten"